guide/14-deep-learning/how_pspnet_works_and_its_implementation_details_within_arcgis.ipynb

```diff
@@ -0,0 +1,234 @@
+{
```

Comment:
<blockquote>The object inside the bounding box in the image is classified as a car(Blue) by an FCN based classifier, which is because it somewhat looks like a car if we view at a local scale, but if the model knew the context in which the object has appeared which is a lake in this case, it would have correctly classified it as a boat(light green)</blockquote><p> </p><p>The boat inside the yellow bounding box is classified as a "car" by a FCN based classifier (c). It is because of its shape and appearance. But it is uncommon to see a car in river.&nbsp;If the model could gain information from the context, for example in our case  water around the object, it will be able to correctly classify it. The PSPNet model (d) is able to capture the context of the whole image to classify the object as  a boat (Green).</p><p> </p><p>Comment: </p><ul><li>I would also suggest providing some evidence to prove that the blue color label corresponds to car and a green color to boat.</li></ul><p> </p><p> </p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/547/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
<p>Done. </p><p>We could add a legend, but I guess, it good this way also.</p>

Reply:
<p>Added legend.</p>